docs/dev/ci/github_actions/adding_tests.md

```diff
@@ -118,7 +118,10 @@ The dedicated workflow example is [`fedora.yml`](./../../../../.github/workflows
 
 ## Test Times and Usage
 
-Be mindful about time and runners usage when adding new steps, jobs and workflows.
+Be mindful about time and runners usage when adding new steps, jobs and workflows. When adding any new test or changing
+the scope of an existing test, always check the test execution time in your PR. As a rule of thumb, the execution time
```

Comment:
```suggestion
the scope of an existing test, always check the test execution time in your pull request (PR). As a rule of thumb, the execution time
```

Reply:
In this place, I just replaced with "pull request", as the acronym is not used in the document. Also made it consistent in other documents.